packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -444,7 +428,7 @@ export class GrpcPlugin extends BasePlugin {
    *  https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/src/plugins/plugin-grpc.ts#L96)
    */
   // tslint:disable-next-line:no-any
-  private getMetadata(original: GrpcClientFunc, args: any[], span: Span):
+  private getMetadata(original: GrpcClientFunc, args: any[]):
```

Comment:
Rather than using `any` here, could this just be an empty object `{}`?

Reply:
Ping on this: @mayurkale22 is it possible to use `{}` here instead of `any` and remove the tslint disable above the line? (This was already here so if you don't want to do it now that's OK, but figure it's nice to clean up if we can while we're in here).